app/controllers/users.js

```diff
@@ -15,7 +15,9 @@ exports.signup = (request, response, next) => {
         firstName: request.body.firstName,
         lastName: request.body.lastName,
         email: request.body.email,
-        password: request.body.password
+        password: request.body.password,
+        is_administrator: false,
```

Comment:
Are you using camelCase or snake_case? Personally i prefer snake_case but the most extended convention in node (and JS in general) is to use camleCase, you are using camelCase in the name too